pyproject.toml

```diff
@@ -39,7 +39,7 @@ sphinx = {version = "^7.0.1", optional = true}
 
 # Poetry has the dependecies groups, but those are not 
 #   compatible with extras, widely used in the python-verse.
-pyrdfstore = {git = "https://github.com/vliz-be-opsci/py-RDF-store.git", rev = "main"}
+pyrdfstore = {git = "https://github.com/vliz-be-opsci/py-RDF-store.git", rev = "fix/issue-51"}
```

Comment:
should we not wait for this to be put into main?

Reply:
indeed -- see own comment about that https://github.com/vliz-be-opsci/py-sync-fs-triples/pull/13#issuecomment-2069102417